server/utils/adjudicationFilterHelper.ts

```diff
@@ -24,12 +24,20 @@ export type UiFilter = {
 
 export const uiFilterFromRequest = (req: Request): UiFilter => {
   return {
-    fromDate: (req.query.fromDate || momentDateToDatePicker(moment().subtract(2, 'days'))) as string,
-    toDate: (req.query.toDate || momentDateToDatePicker(moment())) as string,
+    fromDate: req.query.fromDate as string,
+    toDate: req.query.toDate as string,
     status: req.query.status as ReportedAdjudicationStatus,
   }
 }
 
+export const fillInDefaults = (uiFilter: UiFilter): UiFilter => {
+  return {
+    fromDate: uiFilter.fromDate || momentDateToDatePicker(moment().subtract(2, 'days')),
```

Comment:
Does this cover NN-4081 - Tim did the API (which actually wasn't important) and then put it in for testing?!